vunit/ui/results.py

```diff
@@ -26,3 +31,20 @@ def merge_coverage(self, file_name, args=None):
         """
 
         self._simulator_if.merge_coverage(file_name=file_name, args=args)
+
+    def get_report(self):
+        """
+        Get a report (dictionary) of tests: status, time and output path
+        """
+        return {
+            test.name: {
+                "status": test._status.name,  # pylint: disable=protected-access
```

Comment:
Add a method to_dict to test instead of using private access

Reply:
I added it now. However, private access is required for `_test_results_in_order()`. Should I add `to_dict` to `TestReport` too?